[master < T604-FL] Transactional queue (#768)
memgraph/docs: docs/reference-guide/transactions.md

 - SNAPSHOT ISOLATION
 - READ COMMITTED
 - READ UNCOMMITTED

### Managing transactions

From Memgraph v2.7.0 users can see and terminate certain transactions at a specific moment.

### Show transactions

> To see what transactions are running at a specific moment use the following command:

Reviewer: ```suggestion
To see what transactions are running at the moment use the following command:
```

```cypher
SHOW TRANSACTIONS;
```

The command will show you only transactions you started or which you have the neccessary [privilege](#privileges-needed-to-manage-all-transactions).

<img src={require('../data/how-to-guides/manage-transactional-queue/show_transactions.png').default}/>

### Terminate transactions

To terminate one or more transactions, use the following query:

```cypher
TERMINATE TRANSACTIONS "<tid1>", "<tid2>", "<tid3>" ...
```

> The `tid` represents the transactional ID visible after running the `SHOW TRANSACTIONS;` query.

Reviewer: ```suggestion
The `tid` is the transactional ID that can be seen using the `SHOW TRANSACTIONS;` query.
```

> From the output of `SHOW TRANSACTIONS` command, it is easy to see that an infinite query is being run as part of the transaction with id "9223372036854775809". So in the same session, by executing:

Reviewer: ```suggestion
The output of the `SHOW TRANSACTIONS` command shows that an infinite query is currently being run as part of the transaction ID "9223372036854775809".

To terminate the transaction, run the following query:
```

```cypher
TERMINATE TRANSACTIONS "9223372036854775809";
```

> the user gets the confirmation that the transaction was killed:

Reviewer: ```suggestion
Upon the transaction termination, the following confirmation will appear: 
```

<img src={require('../data/how-to-guides/manage-transactional-queue/terminate_transactions.png').default}/>

> In the session in which infinite query was being run, the user will get a message that the transaction was asked to abort so all changes made as part of that transaction will be annulated and the system will stay in the consistent state as it was before running terminated transaction.

Reviewer: ```suggestion
The following message will appear in the session in which the infinite query was being run:
```
